src/control/server/server.go

```diff
@@ -231,9 +231,12 @@ func Start(log *logging.LeveledLogger, cfg *config.Server) error {
 	// Init management RPC subsystem.
 	mgmtSvc := newMgmtSvc(harness, membership, sysdb, rpcClient, eventPubSub)
 
-	// Forward received events to management service by default.
+	// By default, forward received RASTypeStateChange events to management
+	// service leader to be handled and log RASTypeInfoOnly to INFO.
 	eventForwarder := control.NewEventForwarder(rpcClient, cfg.AccessPoints)
-	eventPubSub.Subscribe(events.RASTypeAny, eventForwarder)
+	eventPubSub.Subscribe(events.RASTypeStateChange, eventForwarder)
+	eventLogger := control.NewEventLogger(log)
+	eventPubSub.Subscribe(events.RASTypeInfoOnly, eventLogger)
```

Comment:
I'm not sure this is correct. Wouldn't it mean that state change events aren't logged? It's true that some events should only be logged and not forwarded, but all events should be logged.

There is also an open question (in my mind, at least) about where the forwarded events should be logged. Should they be logged locally, on the receiving server, or at both locations? I think to start with, we should just log them locally before forwarding them.